ciphers/rsa_factorization.py

```diff
@@ -5,15 +5,13 @@
 public key e.
 Source: on page 3 of https://crypto.stanford.edu/~dabo/papers/RSA-survey.pdf
 More readable source: https://www.di-mgt.com.au/rsa_factorize_n.html
-large number can take minutes to factor, therefore are not included in doctest.
 """
-from __future__ import annotations
-
 import math
 import random
+from typing import List
 
 
-def rsafactor(d: int, e: int, N: int) -> list[int]:
+def rsafactor(d: int, e: int, N: int) -> List[int]:

```

Comment:
```suggestion
def rsa_factor(d: int, e: int, N: int) -> tuple[int, int]:
```
I think it'd be better for the function to return `p` and `q` as a tuple because it'd allow users to use tuple unpacking with the return values:
```py
from ciphers.rsa_factorization import rsa_factor

...

p, q = rsa_factor(d, e, N)
# do something with p and q
```

(Also the function name should be in snake case)